tests/pytests/unit/modules/test_chroot.py

```diff
@@ -0,0 +1,345 @@
+#
+# Author: Alberto Planas <aplanas@suse.com>
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+:maintainer:    Alberto Planas <aplanas@suse.com>
+:platform:      Linux
+"""
+
+import sys
+
+import pytest
+
+import salt.loader.context
+import salt.modules.chroot as chroot
+import salt.utils.platform
+from salt.exceptions import CommandExecutionError
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.mark.skipif(
+    salt.utils.platform.is_windows(), reason="This test cannot work on Windows"
+)
```

Comment:
```suggestion
pytestmark = [
    pytest.mark.skipif(
        salt.utils.platform.is_windows(), reason="This test cannot work on Windows"
    )
]
```

This should add a global skip.

Reply:
thanks